acceptance/suites/tests/010-puppetserver-cli/subcommand/gem.rb

```diff
@@ -14,7 +14,7 @@
   gem_install += " --clear-sources --source #{additional_gem_source}"
 end
 
-gem_uninstall = "#{cli} gem uninstall"
+gem_uninstall = "#{cli} gem uninstall --silent"
```

Comment:
Oh did this break on redhat too? Is it java 11 related?

Reply:
Yeah, seems java 11 related. Didn't find out what was causing it, but passing silent seemed more correct anyways.